recipes/idr-py/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "idr-py" %}
+{% set version = "0.2.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "af50a517b2fa324a1030830d95803cda18ecf2f9381550f401651f8c25d2f322" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  skip: true  # [py3k]
```

Comment:
[Guidelines](https://bioconda.github.io/guidelines.html#python) recommend:
```yaml
  skip: True  # [not py27]
```

Reply:
Thanks, @nsoranzo. Unfortunately seeing the same error with this change:

```
/opt/conda/lib/python3.6/site-packages/idr/tests/test_attributes.py
```

(I had tried various variants of this and force pushed them away)